src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -42,18 +42,6 @@ public function __construct($throwOnAutowireException = true)
     }
 
     /**
-     * @deprecated since version 3.4, to be removed in 4.0.
-     *
-     * @return AutowiringFailedException[]
-     */
-    public function getAutowiringExceptions()
-    {
-        @trigger_error('Calling AutowirePass::getAutowiringExceptions() is deprecated since Symfony 3.4 and will be removed in 4.0. Use Definition::getErrors() instead.', E_USER_DEPRECATED);
-
-        return $this->autowiringExceptions;
```

Comment:
the property should be removed also (same in the other pass)

Reply:
Good catch. Done.